packages/story-editor/src/components/colorPicker/editablePreview.js

```diff
@@ -131,11 +131,14 @@ function EditablePreview({ label, value, width, format, onChange }) {
     );
   }
 
+  // The value is set to 'transparent' by the library if it's black with 0 opacity.
```

Comment:
Is that the underlying `react-color` library, that does this? Is that a breaking change?

Reply:
Yep, that's the underlying `react-color`, it displays `transparent` by default if it's `000000` and opacity `0`. Shouldn't be a breaking change, all seems to work well. If that's what you mean :)